### DISCUSSION TITLE
File attributes: How to upload an attachment to a custom entity file attribute

### DISCUSSION THREAD

**Discussion by nrpieper** [AymericM78/PowerDataOps#3] [Q&A]
CDS now supports file type columns.  I know the record needs to be created first before being able to add binary content to the file attribute, but I'm not sure how to construct the command.

Any help would be appreciated.

**Comment by AymericM78** [comment]
Hello @nrpieper !
PowerDataOps cmdlet is just a pshell wrapper of Microsoft Dataverse SDK (aka CDS), so if the sdk provide functions to interact as you expect, we can add it as a cmdlet.

Regarding your ask, I found the following documentation: 
https://docs.microsoft.com/en-us/powerapps/developer/data-platform/file-attributes

Hope this help!
